build.gradle

```diff
@@ -27,8 +27,8 @@ def assetsDir = new File(projectDir, "src/main/resources/assets")
 
 compass {
     importPath = files(
-        new File(assetsDir, "govuk_frontend_toolkit/stylesheets"),
-        new File(assetsDir, "govuk_template/source/assets/stylesheets")
+            new File(assetsDir, "govuk_frontend_toolkit/stylesheets"),
```

Comment:
Is this indentation right? 2 spaces?

Reply:
We don't have two spaces in gradle, we have 1 tab (4 spaces) but in this case is something embedeed, for this reasons is 2 tabs.